### DISCUSSION TITLE
Stream json lines from api route

### DISCUSSION THREAD

**Discussion by snorkypie** [vercel/next.js#17796] [Help] [answered]
I can't seem to figure out how to "stream" data back to the client from an api route. In its basic form I'd like to just do res.json() multiple times. Any tips?

In this exact example it's progress of a file upload and then progress of the processing of that same file.

**Comment by ijjk** [comment, marked as answer]
Hi, this comment might be what you are looking for https://github.com/vercel/next.js/issues/9965#issuecomment-584319868

**Reply by snorkypie** [reply]
Oh yeah, that did it! Big thanks!
